### ISSUE TITLE
Not showing results from specific indexer. Works on Sonarr.

### ISSUE THREAD

**Issue by gilbN** [Radarr/Radarr#2127]
I just noticed that Radarr is not showing any results from one of my indexers (https://danishbits.org)
I use Jackett to connect to it, and **Sonarr** shows results just fine. The test connection works in Radarr and Jackett.  I have also tested Cardigann, no results there.  But manual searching works on both Jackett and Cardigann
**Radarr Version:** 0.2.0.852
 **Mono Version** 5.2.0.22
**Logs:**

DanishBits HTTP request failed: [502:BadGateway] [GET] at [http://192.168.1.34:5060/torznab/danishbits/api?t=movie&cat=2000,2010,2020,2030,2035,2040,2045,2050,2060&extended=1&apikey=(removed)&offset=0&limit=100&imdbid=0112864]
Close

HTTP Error - Res: [GET] http://192.168.1.34:5060/torznab/danishbits/api?t=movie&cat=2000,2010,2020,2030,2035,2040,2045,2050,2060&extended=1&apikey=(removed)&offset=0&limit=100&imdbid=0112864: 502.BadGateway
<error>
  <code>900</code>
  <description>Expected to find `.title_wrapper h1`</description>
</error>

**Comment by hotio** [comment]
What makes you think that if it works in Sonarr, it should work in Radarr, Sonarr is for tv shows, radarr is for movies, two totally different animals.

What happens when you drop `imdbid=0112864` from the url?

**Comment by gilbN** [comment]
How do you mean? imdbid=0112864 is something radarr added when I searched for a movie.

**Comment by hotio** [comment]
Yes, I know!

Let me rephrase. What happens when you visit the url
```
http://192.168.1.34:5060/torznab/danishbits/api?t=movie&cat=2000,2010,2020,2030,2035,2040,2045,2050,2060&extended=1&apikey=(FILL IN API kEY)&offset=0&limit=100
```

**Comment by gilbN** [comment]
Then it works, it lists results from the site!

**Comment by gilbN** [comment]
if I add imdbid=tt0112864

I get this

`<rss xmlns:torznab="http://torznab.com/schemas/2015/feed" version="2.0">
  <channel>
    <title>DanishBits</title>
    <link>https://danishbits.org</link>
    <language>da-dk</language>
  </channel>
</rss>`

**Comment by thamilton91** [comment]
I'm also experiencing this issue with torznab through cardigann using TorrentDay.

Looks like if I remove the imdbid from the URL it'll work. Will there be a way to have this removed from torznab searching?

**Comment by hotio** [comment]
Cardigann is as good as dead last I saw, they just need to pull the plug...

**Comment by thamilton91** [comment]
Is there something else that is better?

**Comment by gilbN** [comment]
Jackett perhaps?

**Comment by thamilton91** [comment]
I went with cardigann because I was under the impression jackett was dead lol

**Comment by hotio** [comment]
I would not call multiple commits a day, dead.

**Comment by gilbN** [comment]
I was able to add the tracker using Torrentpotato in radarr

**Comment by hotio** [comment]
Do not use torrentpotato, pick torznab

**Comment by hotio** [comment]
It seems cardigann has had quite a few commits recently....doubt...

I would certainly try Jackett with torznab, that should work 100% (mostly)

**Comment by thamilton91** [comment]
Jackett renders the same problem. 

I did try using torrentpotato like @gilbN suggested. That works actually.

**Comment by hotio** [comment]
I hope somebody created a issue for this on the relevant github project, otherwise it's gonna take a while... Since the url does not return data when imdb is added, this has nothing to do with radarr. I think ™®©

**Comment by thamilton91** [comment]
@hotio is it possible to set Raddar to not use "IMDBID" when searching torznab?

**Comment by hotio** [comment]
Not that I know

**Comment by galli-leo** [comment]
@thamilton91 That is now possible. Also torrentpotato is deprecated and will receive no further support, please switch to Torznab.
